modules/rest-api/src/main/java/org/wso2/carbon/mdm/api/Operation.java

```diff
@@ -93,8 +91,9 @@ public Message addOperation(DeviceOperationContext operationContext) throws MDMA
 	}
 
     @POST
-    @Path("installApp")
-    public Message installApplication(ApplicationWrapper applicationWrapper) throws MDMAPIException {
+    @Path("installApp/{tenantDomain}")
```

Comment:
Is it required to have tenant domain at the request path it self? Also, it's not recommendable to have a verb like "installApp" to name a REST Resource path.